### PR TITLE
QChem: parse atomcoords in bohr

### PR DESCRIPTION
**Pull request by berquist**
This PR should go in before https://github.com/cclib/cclib-data/pull/127.

### CODE REVIEW

File: cclib/parser/qchemparser.py [hunk 1]
```diff
@@ -691,20 +691,26 @@ def extract(self, inputfile, line):
                 self.append_attribute('time', float(tokens[8]))
 
             # Extract the atomic numbers and coordinates of the atoms.
-            if 'Standard Nuclear Orientation (Angstroms)' in line:
-                if not hasattr(self, 'atomcoords'):
-                    self.atomcoords = []
+            if 'Standard Nuclear Orientation' in line:
+                if "Angstroms" in line:
+                    def convertor(x):
```
Comment (langner) [inline cclib/parser/qchemparser.py:696]: How about `convertor = lambda x: x` and `convertor = utils.convertot(...)` below?

File: cclib/parser/qchemparser.py [hunk 2]
```diff
@@ -691,20 +691,26 @@ def extract(self, inputfile, line):
...
+                        return x
+                elif 'Bohr' in line:
+                    def convertor(x):
+                        return utils.convertor(x, 'bohr', 'Angstrom')
+                else:
+                    raise RuntimeError("Unknown units in coordinate header: {}".format(line))
```
Comment (langner) [inline cclib/parser/qchemparser.py:702]: Probably `ValueError` is more appropriate.

File: cclib/parser/qchemparser.py (hunk 1 above)
Comment (berquist) [reply cclib/parser/qchemparser.py]: Next time I'll listen to my gut rather than the linter.

File: cclib/parser/qchemparser.py [hunk 3]
```diff
@@ -691,20 +691,24 @@ def extract(self, inputfile, line):
...
+                    raise ValueError("Unknown units in coordinate header: {}".format(line))
                 self.skip_lines(inputfile, ['cols', 'dashes'])
                 atomelements = []
                 atomcoords = []
                 line = next(inputfile)
                 while list(set(line.strip())) != ['-']:
                     entry = line.split()
                     atomelements.append(entry[1])
-                    atomcoords.append(list(map(float, entry[2:])))
+                    atomcoords.append([convertor(float(value)) for value in entry[2:]])
```
Comment (shivupa) [inline cclib/parser/qchemparser.py:708]: This `convertor` lambda stuff is great

**Pull request merged**